docs/source/Learn/bskPrinciples/bskPrinciples-6.rst

```diff
@@ -1,8 +1,3 @@
-.. raw:: html
-
-    <iframe width="560" height="315" src="https://www.youtube.com/embed/XuaSmG4wYlk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Note that, unfortunately, the changes mean that this video tutorial is outdated.

Reply:
Yeah, I'll have to re-record a new video for this.  I'll do this after we wrap up the PR review.